Marlin/Marlin_main.cpp

```diff
@@ -5581,7 +5583,7 @@ void home_all_axes() { gcode_G28(true); }
         zero_std_dev_old = zero_std_dev;
         zero_std_dev = round(SQRT(S2 / N) * 1000.0) / 1000.0 + 0.00001;
 
-        // Solve matrices
+        // Solve matrices (see https://github.com/LVD-AC/Marlin-AC/tree/1.1.x-AC/documentation)
```

Comment:
Or copy this file to some place safe and refer to that location (allow me to edit it if needed)